graspologic/utils/utils.py

```diff
@@ -354,14 +365,20 @@ def to_laplace(graph, form="DAD", regularizer=None):
            [0.70710678, 0.        , 0.        ]])
 
     """
+
     valid_inputs = ["I-DAD", "DAD", "R-DAD"]
     if form not in valid_inputs:
         raise TypeError("Unsuported Laplacian normalization")
 
     A = import_graph(graph)
 
-    in_degree = np.sum(A, axis=0)
-    out_degree = np.sum(A, axis=1)
+    # def _sparse(A, form, regularizer):
+    if isspmatrix_csr(A):
+        in_degree = np.array(graph.sum(axis=0), dtype=np.float64)[0]
```

Comment:
check if the casting is different between scipy/numpy 

use A.sum() in either case? (i think)